ethereum_history_api/circuits/lib/src/fixtures/paris/usdc/storage_proof.nr

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
The file should not start with an empty line. Otherwise VsCode removes it as long as you click save within it

Reply:
fixed